docs/hosting-packages/Overview.md

```diff
@@ -22,7 +22,7 @@ There are also several other NuGet hosting products such as [Azure Artifacts](ht
 
 - [Artifactory](https://www.jfrog.com/artifactory/) from JFrog.
 - [Azure Artifacts](https://www.visualstudio.com/docs/package/nuget/publish), which is also available on Team Foundation Server 2017 and later.
-- [BaGet](https://github.com/loic-sharma/BaGet), an open-source implementation of NuGet V3 server built on ASP.NET Core
+- [BaGetter](https://github.com/bagetter/BaGetter), an open-source and community driven implementation of NuGet V3 server built on ASP.NET Core
```

Comment:
Team Review:  This pull request can add BaGetter but we'd prefer the owner of BaGet to ask to have it removed  @loic-sharma

Reply:
I haven't discontinued BaGet, however, the last commit was 2 years ago. 

The BaGetter fork is active, but it's a fairly new fork from what I can tell. I can't speak to the quality/security of this fork.

Personally, I would list both options and let folks choose.